modules/core/shared/src/main/scala/lucuma/core/model/Proposal.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.Order._
+import cats.syntax.all._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.enum.TacCategory
+import lucuma.core.enum.ToOActivation
+import lucuma.core.math.units.IntPercent
+import monocle.Focus
+
+import scala.collection.immutable.SortedMap
+
+final case class Proposal(
+  title:         Option[NonEmptyString],
+  proposalClass: ProposalClass,
+  category:      Option[TacCategory],
+  toOActivation: ToOActivation,
+  abstrakt:      Option[NonEmptyString],
+  partnerSplits: SortedMap[Partner, IntPercent]
+)
+
+object Proposal {
+  val title         = Focus[Proposal](_.title)
+  val proposalClass = Focus[Proposal](_.proposalClass)
+  val category      = Focus[Proposal](_.category)
+  val toOActivation = Focus[Proposal](_.toOActivation)
+  val abstrakt      = Focus[Proposal](_.abstrakt)
+  val partnerSplits = Focus[Proposal](_.partnerSplits)
+
+  val Default: Proposal =
+    Proposal(None, ProposalClass.Queue(80), None, ToOActivation.None, None, SortedMap.empty)
+
+  implicit val eqProposal: Eq[Proposal] = Eq.instance {
```

Comment:
Maybe `Eq.by`?

Reply:
I can switch to that. It is shorter. But, the thing I don't like about `Eq.by` is that it is easy to forget to add fields as the class changes (which I'm assuming will happen here). I tried this approach because it is impossible for that to happen, but it is definitely more verbose.

Reply:
Do you mean `EqTests[Proposal].eqv`? I have that, but I think `Eq` based on a subset of fields is a valid instance wrt the tests. It just isn't what we would usually want. I'm looking forward to someday being able to derive Eq so that this isn't an issue.

Reply:
There is an urban legend that derivation is working fairly well in scala 3